package.json

```diff
@@ -170,6 +171,10 @@
     "*.js.map"
   ],
   "exports": {
+    ".": {
+      "types": "./index.d.ts",
+      "default": "./index.js"
+    },
```

Comment:
Auto-complete / adding missing imports seems to work well. Both IntelliJ and VS Code automatically add an import from `@vaadin/react-components` when auto-completing a component name. If a component import is missing both suggest importing first from the index file, and then from the component specific file.

Had some hickups in IntelliJ with some individual components, but that might be some cache issue - can look further into this later.

Reply:
Looks like most issues in IntelliJ are gone after clearing the cache. The only one that remains is that it suggests two import options for `Grid`, one of which is invalid:
<img width="537" alt="Bildschirmfoto 2024-01-17 um 10 56 20" src="https://github.com/vaadin/react-components/assets/357820/ce293b44-91b2-414d-8f1b-005a5006341b">

Reply:
There is a ticket now: #224